### ISSUE TITLE
Move demo off of Heroku

### ISSUE THREAD

**Issue by ajmeese7** [ajmeese7/readme-ascii#14]
As we all know, Heroku is getting rid of its free tier. I need to migrate the project demo to another site so hosting can continue unimpeded.

A good list of services to consider instead can be found [here](https://github.com/Engagespot/heroku-free-alternatives).

**Comment by lewxdev** [comment]
Heroku deployment is currently down for this. Perhaps Fly could be a fair alternative here?

**Comment by ajmeese7** [comment]
Definitely, I'll try to do that in the next few days! Will update with the results.

**Comment by BedBug2479** [comment]
> Definitely, I'll try to do that in the next few days! Will update with the results.

hi the demo is still down
will there be an ETA on if the site is revived?

**Comment by ajmeese7** [comment]
@BedBug2479 there is not, life got super busy and I haven't had the time to do much open source work in a while. I'll try to get that spun up today!

Update: haven't got it fully working and live yet, but my work is taking place in https://github.com/ajmeese7/readme-ascii/tree/fly-transition.

**Comment by ajmeese7** [comment]
@BedBug2479 just changed this over to being a static site, it should work much better now!
